src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/GetAzureRmRecoveryServicesBackupJob.cs

```diff
@@ -117,6 +117,10 @@ public override void ExecuteCmdlet()
                 {
                     throw new Exception(Resources.JobAllowedDateTimeRangeExceeded);
                 }
+                else if(rangeStart > DateTime.UtcNow)
```

Comment:
You have roughly 30 lines of validation code for the dates. Do you any tests that gives you a decent coverage for your date validation logic?